### DISCUSSION TITLE
2021-09-08 Weekly Meeting 20 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#86] [Weekly Meetings]
Weekly Meeting 20 is happening on Wednesday September 8, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - [OMI Survey](https://forms.gle/Dxvap4auBGim6JsL9) follow-up
   - [glTF Sub Group](https://github.com/omigroup/gltf-extensions) follow-up
   - [Retrospective](https://app.mural.co/t/altontech5044/m/altontech5044/1630338172367/94b4adf757f3a4842bf002a6c400c2b94ec99527)  follow-up
   - Demo Night coordination follow-up
     - https://forms.gle/CknpQQKNi7ywQu9u5
   - OMI Website follow-up

Add ons: 
   - Discord Roles/Bot follow-ups
   - Working agreements / [playbook lite template](https://app.mural.co/t/altontech5044/m/altontech5044/1630340368839/d4029b3d2dd4efae2dc124995842fe70253a2bfa)

**Comment by lyuma** [comment]
I would like to have a conversation if OMI would like representation at Virtual Market on VRChat. The deadline for applying is this week.

https://mobile.twitter.com/Virtual_Market_/status/1431481743895187461
https://winter2021.vket.com/
